### ISSUE TITLE
Relay server doesn't work (Unhandled Runtime Error Error: Hydration failed because the initial UI does not match what was rendered on the server.)

### ISSUE THREAD

**Issue by Ayush-a3h** [run-llama/voice-chat-pdf#3]
Unhandled Runtime Error
Error: Hydration failed because the initial UI does not match what was rendered on the server.
![image](https://github.com/user-attachments/assets/cdcd4485-ffe1-483e-936c-9a465127bc4d)

## Steps to Reproduce:

1. Clone the repository.
2. Add your OpenAI API key and REACT_APP_LOCAL_RELAY_SERVER_URL=http://localhost:8081  to the `.env` file.
3. Run the following commands simultaneously:
   - `npm install`
   - `npm run dev`
   - `npm run relay`

**Comment by marcusschiesser** [comment]
yes that one is not implemented yet - needs to be moved to nextjs
